### PR TITLE
feat(language-selector): add vuestic-language-selector component

### PR DESCRIPTION
**Pull request by yevdyko**
This PR adds `language-selector` component, includes it to navbar and removes `Change Language` section from `User Profile` dropdown in navbar. It uses `flag-icon-css` library for country flags in SVG.

### CODE REVIEW

File: src/components/vuestic-components/vuestic-language-selector/VuesticLanguageSelector.vue [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        <a
+          class="dropdown-item"
+          href="#"
+          v-for="(option, id) in options"
+          :key="id"
+          @click="setLanguage(option.code)"
+        >
+          <i class="i-flag-small"></i>
+          <span class="dropdown-item-text ellipsis">
+            {{ option.name }}
```
Comment (yevdyko) [inline src/components/vuestic-components/vuestic-language-selector/VuesticLanguageSelector.vue]: @smartapant How to localize a text inside the expression `{{ ... }}`?

**Comment by smartapant** [comment]
@yevdyko language selection is empty on initial page load. Also, selected language shall be green when dd is shown

**Comment by yevdyko** [comment]
@smartapant updated.

**Comment by yevdyko** [comment]
@smartapant Should we use a fixed dropdown width or relative depending on the item text?

**Comment by smartapant** [comment]
@yevdyko you could use min/max widths. As for mobile view - asked the designer. Will update you

**Pull request merged**